packages/react/src/components/UIShell/SideNavItems.tsx

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright IBM Corp. 2016, 2023
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import cx from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { CARBON_SIDENAV_ITEMS } from './_utils';
+import { usePrefix } from '../../internal/usePrefix';
+
+interface SideNavItemsProps {
+  /**
+   * Provide an optional class to be applied to the containing node
+   */
+  className?: string;
+  /**
+   * Provide a single icon as the child to `SideNavIcon` to render in the
+   * container, and it is required
+   */
+  children: React.ReactNode;
+  /**
+   * Property to indicate if the side nav container is open (or not). Use to
+   * keep local state and styling in step with the SideNav expansion state.
+   */
+  isSideNavExpanded?: boolean;
```

Comment:
Just some formatting pieces here

```suggestion
  /**
   * Provide a single icon as the child to `SideNavIcon` to render in the
   * container
   */
  children: React.ReactNode;

  /**
   * Provide an optional class to be applied to the containing node
   */
  className?: string;

  /**
   * Property to indicate if the side nav container is open (or not). Use to
   * keep local state and styling in step with the SideNav expansion state.
   */
  isSideNavExpanded?: boolean;
```

Reply:
i'll keep formatting in mind before making a PR.